src/shutdown/shutdown.c

```diff
@@ -389,8 +389,8 @@ int main(int argc, char *argv[]) {
         umask(0022);
 
         if (getpid_cached() != 1) {
-                r = log_error_errno(SYNTHETIC_ERRNO(EPERM), "Not executed by init (PID 1).");
-                goto error;
+                log_error("Not executed by init (PID 1). Refusing to operate.");
+                return EXIT_FAILURE;
```

Comment:
Then, maybe, move the check earlier than parse_argv()? Otherwise, freeze() will be called if we manually invoke it with invalid arguments.